### PR TITLE
feat(vm-logs): AM/PM

### PR DESCRIPTION
**Pull request by wowshakhov**

### CODE REVIEW

File: src/app/shared/services/utils/time-examples.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
+export default [
```
Comment (tamazlykar) [inline src/app/shared/services/utils/time-examples.ts]: Is this worth to be in a different file?

File: src/app/shared/services/utils/utils.service.spec.ts [hunk 2]
```diff
@@ -125,4 +126,33 @@ describe('Utils service', () => {
...
   });
+
+  it('should convert AM/PM to 24 and 24 to AM/PM', () => {
+    timeExamples.forEach(({ hour12, hour24 }) => {
+      expect(Utils.convertAmPmTo24(hour12)).toEqual(hour24);
+      expect(Utils.convert24ToAmPm(hour24)).toEqual(hour12);
+    });
+
+    expect(
+      Utils.convert24ToAmPm({
```
Comment (tamazlykar) [inline src/app/shared/services/utils/utils.service.spec.ts]: If you have a dataset array, why you did not put this test cases in datasets array?

File: src/app/shared/services/utils/utils.service.spec.ts [hunk 3]
```diff
@@ -125,4 +126,33 @@ describe('Utils service', () => {
     color = '#000000';
     expect(Utils.isColorDark(color)).toBeTruthy();
   });
+
+  it('should convert AM/PM to 24 and 24 to AM/PM', () => {
+    timeExamples.forEach(({ hour12, hour24 }) => {
+      expect(Utils.convertAmPmTo24(hour12)).toEqual(hour24);
+      expect(Utils.convert24ToAmPm(hour24)).toEqual(hour12);
```
Comment (tamazlykar) [inline src/app/shared/services/utils/utils.service.spec.ts]: Maybe better to separate tests for each function?

**Comment by wowshakhov** [comment]
To test: 
Time inputs in recurring snapshots (regression)

**Pull request merged**